package.json

```diff
@@ -25,6 +25,7 @@
     "babel-register": "^6.3.13",
     "boom": "^2.8.0",
     "csv-stringify": "0.0.8",
+    "date-fns": "^1.28.5",
```

Comment:
We're already using moment.js throughout the project, can we use that here as well to avoid another library.

Reply:
Ah, I didn't see `moment` in the package.json, but I could switch to it.

Reply:
Yes, please! Would be nice to limit packages across the projects.